test/Microsoft.AspNet.Mvc.TagHelpers.Test/TestableHtmlGenerator.cs

```diff
@@ -82,7 +83,7 @@ public override HtmlString GenerateAntiforgery(ViewContext viewContext)
                 },
             };
 
-            return tagBuilder.ToHtmlString(TagRenderMode.SelfClosing);
+            return new HtmlString(tagBuilder.ToHtmlContent(TagRenderMode.SelfClosing).ToString());
```

Comment:
This looks incorrect. Why not return an `IHtmlContent` written using the `CommonTestEncoder`?